pkg/core/image_client.go

```diff
@@ -55,7 +55,7 @@ type ListImagesOptions struct {
 	Manifest string
 	Images   string
 	Force    bool
-	Check    bool
+	NoCheck  bool
```

Comment:
Not sure about changing this variable. It's slightly easier to reason about `Check` being true or false rather than `NoCheck`. In general, I think the command layer should deal with the UX and the core layer should be biassed towards the implementors.

Reply:
I fully agree with that and I'm not of double negation either...
However, I've had trouble to bind a cobra flag to a simple bool variable (or even better, have a custom binding that would negate the flag value directly into the option struct field).

Reply:
Ah, I see your predicament. Let's stick with `NoCheck` then! :-)